huawei_vrp/docker/launch.py

```diff
@@ -164,19 +218,43 @@ def bootstrap_config(self):
         self.wait_write(
             cmd=f"ssh user {self.username} authentication-type password ", wait="]"
         )
-        self.wait_write(cmd=f"ssh user {self.username} service-type all ", wait="]")
+        if self.vm_type != "AR1000V":
+            self.wait_write(cmd=f"ssh user {self.username} service-type all ", wait="]")
         self.wait_write(cmd="stelnet server enable", wait="]")
 
         # NETCONF
-        self.wait_write(cmd="snetconf server enable", wait="]")
-        self.wait_write(cmd="netconf", wait="]")
-        self.wait_write(cmd="protocol inbound ssh port 830", wait="]")
-        self.wait_write(cmd="quit", wait="]")
-
-        self.wait_write(cmd="commit", wait="]")
-        self.wait_write(cmd="return", wait="]")
-        self.wait_write(cmd="save", wait=">")
-        self.wait_write(cmd="undo mmi-mode enable", wait=">")
+        if self.vm_type != "AR1000V":
+            self.wait_write(cmd="snetconf server enable", wait="]")
+            self.wait_write(cmd="netconf", wait="]")
+            self.wait_write(cmd="protocol inbound ssh port 830", wait="]")
+            #self.wait_write(cmd="quit", wait="]")
+
+            # Envia o comando commit inicialmente
+            self.wait_write(cmd="commit", wait="]")
+
+            while True:
+                (ridx, match, res) = self.tn.expect([
+                    b"Error: The system is busy in building system configurations. Please wait a moment and then try again.",  # 3
+                ], timeout=1)
+
+                if match:
+                    if ridx == 0:  # Mensagem de erro ao tentar commit
+                        print("Sistema ocupado, aguardando 5 segundos para tentar novamente...")
```

Comment:
Same here about language. Also, don't `print` to stdout, but rather use the logger.